docs/products/ESPHome-Starter-Kit/modules/temperature-humidity-module.md

```
@@ -125,4 +125,14 @@ With the device back online, the temperature and humidity entities are live on t
 
 !!! success "Your Temperature and Humidity module is wired up"
 
-    You now have a trustworthy temp and humidity sensor to place anywhere in your home!
\ No newline at end of file
+    You now have a trustworthy temp and humidity sensor to place anywhere in your home!
+
+## Try it in an automation
+
+Your sensor is reporting. Now make it drive something:
+
+<a href="/products/ESPHome-Starter-Kit/automations/temp-reactive-leds/" class="md-button md-button--primary"> <img src="/assets/esphome-logo.svg" /> Temp-Reactive LEDs </a>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)